src/main/java/com/rb/nonbiz/types/RandomNumberGeneratorSeed.java

```diff
@@ -0,0 +1,29 @@
+package com.rb.nonbiz.types;
+
+import com.rb.nonbiz.util.RBPreconditions;
+
+/**
+ * Just an int, but with the semantics that it will be used as a seed in a random number generator.
+ *
+ * <p> This may be overly restrictive, but we only allow non-negative seeds. </p>
+ */
+public class RandomNumberGeneratorSeed extends IntegerValue<RandomNumberGeneratorSeed> {
```

Comment:
I'm surprised that we didn't have this 8+ years into this. I suppose we never needed to deal with random numbers so far.